Source/bmalloc/libpas/src/libpas/bmalloc_heap_inlines.h

```diff
@@ -23,8 +23,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
  */
 
-#ifndef BMALLOC_HEAP_INLINES_H
-#define BMALLOC_HEAP_INLINES_H
+#pragma once
```

Comment:
Please revert this.  bmalloc is meant to be a C project that can be #included in C files, and we would like to retain that until we have a strong reason not to.